eras/conway/impl/src/Cardano/Ledger/Conway/Governance.hs

```diff
@@ -203,34 +212,119 @@ instance EraPParams era => ToCBOR (GovActionsState era) where
 instance EraPParams era => FromCBOR (GovActionsState era) where
   fromCBOR = fromEraCBOR @era
 
+data LastGovActions era = LastGovActions
```

Comment:
What do you guys think about this name? I initially called it `PrevGovActions`, but then I thought that having this as "last" seems more to indicate the "state"  of the system as opposed to the `Prev` reference from the proposals. But maybe it's creating more confusion.. I'm happy to change it to `PrevGovActions` or anything else if anyone thinks that's better.

Reply:
I agree,  happy with `PrevGovActionIds`, thanks :+1: